### PR TITLE
DOP-2451 Configure 10gen/docs-mongodb-internal to be buildable

### PR DESCRIPTION
**Pull request by GuruPKK**

### CODE REVIEW

File: makefiles/Makefile.docs-mongodb-internal [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+
+include ~/shared.mk
+
+
+get-build-dependencies: 
+	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-mongodb-internal.yaml > ${REPO_DIR}/published-branches.yaml
```
Comment (i80and) [inline makefiles/Makefile.docs-mongodb-internal:21]: Ideally, the redirection target should be quoted just in case REPO_DIR contains spaces

**Review by i80and — APPROVED** [review]

**Comment by jeff-allen-mongo** [comment]
@GuruPKK , we would like this repo to be buildable with next-gen but explicitly _not publishable_. Is this doable? Sorry for not explicitly stating this in the ticket!

**Comment by GuruPKK** [comment]
> @GuruPKK , we would like this repo to be buildable with next-gen but explicitly _not publishable_. Is this doable? Sorry for not explicitly stating this in the ticket!

Sure, just curious if it is not published, how can we verify whether the build worked?

**Comment by jeff-allen-mongo** [comment]
The workflow we'd like for this repo is:

1. Writers fork the repo.
2. Writers push changes to their origin branch.
3. The autobuilder gets triggered. Writers receive a build log / staging URL to verify changes.

We'd just like to use this repo as a way to view staged changes and do reviews, but do not want its contents to be available to external users.

**Pull request merged**